### PR TITLE
Vietnamese translation

### PR DESCRIPTION
**Pull request by shortthu**
~~WIP stuff, will update until it done.~~

Progress might be slow because I'm doing both translating and reviewing in my fork, also we are not free all the time 😛

### CODE REVIEW

File: resources/lang/vi/admin.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        'orders' => [
+            'index' => [
+                'title' => 'Danh Sách Đơn Hàng',
+            ],
+        ],
+    ],
+
+    'users' => [
+        'restricted_banner' => [
+            'title' => 'Người dùng đang bị restricted.',
```
Comment (shortthu) [inline resources/lang/vi/admin.php]: @AcmeDVoid sao chỗ này không để "Người dùng đang bị hạn chế" luôn :3c
//blame shortthu for not pointing this out on pr

**Comment by shortthu** [comment]
back again \ :D /

**Comment by peppy** [comment]
What's the status on this one?

**Comment by shortthu** [comment]
Sorry for being slow, I'm still on it

File: resources/lang/vi/admin.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+return [
+    'beatmapsets' => [
+        'covers' => [
+            'regenerate' => 'Phục hồi',
+            'regenerating' => 'Đang phục hồi...',
+            'remove' => 'Gỡ bỏ',
+            'removing' => 'Đang gỡ bỏ...',
+        ],
+        'show' => [
+            'covers' => 'Quản lý Các ảnh cover của Beatmapset',
```
Comment (Gagako) [inline resources/lang/vi/admin.php]: ảnh bìa?

File: resources/lang/vi/admin.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'beatmapsets' => [
+        'covers' => [
+            'regenerate' => 'Phục hồi',
```
Comment (Gagako) [inline resources/lang/vi/admin.php]: mà phục hồi ở đây là cái gì thế

Comment (shortthu) [reply resources/lang/vi/admin.php]: như trong code thì đây là phục hồi ảnh bìa

**Comment by peppy** [comment]
Is this one still WIP or are we ready to push it out?

**Comment by shortthu** [comment]
Almost ready, just need a little bit longer (for new strings and checks)

**Comment by peppy** [comment]
Please do not PR from your master branch ever again.

**Pull request merged**